docs/tables/aws_cost_optimization_recommendation/index.md

```diff
@@ -0,0 +1,212 @@
+
+```sh
+vi ~/.tailpipe/config/aws.tpc
+```
+
+```hcl
+connection "aws" "billing_account" {
+  profile = "my-billing-account"
+}
+
+partition "aws_cost_optimization_recommendation" "my_recommendations" {
+  source "aws_s3_bucket" {
+    connection = connection.aws.billing_account
+    bucket     = "aws-cost-optimization-recommendations-bucket"
+  }
+}
+```
+
+## Collect
+
+[Collect](https://tailpipe.io/docs/manage/collection) data for all `aws_cost_optimization_recommendation` partitions:
+
+```sh
+tailpipe collect aws_cost_optimization_recommendation
+```
+
+Or for a single partition:
+
+```sh
+tailpipe collect aws_cost_optimization_recommendation.my_recommendations
+```
+
+## Query
+
+**[Explore 10+ example queries for this table →](https://hub.tailpipe.io/plugins/turbot/aws/queries/aws_cost_optimization_recommendation)**
+
+### Top Cost-Saving Recommendations
+
+Identify the recommendations with the highest potential monthly savings.
+
+```sql
+select
+  recommendation_id,
+  account_id,
+  current_resource_type,
+  recommended_resource_type,
+  region,
+  estimated_monthly_savings_after_discount as savings_amount,
+  estimated_savings_percentage_after_discount as savings_percentage,
+  currency_code
+from
+  aws_cost_optimization_recommendation
+order by
+  savings_amount desc
+limit 10;
+```
+
+### Easy to Implement Recommendations
+
+Find recommendations that are easy to implement and don't require restarts.
+
+```sql
+select
+  recommendation_id,
+  account_id,
+  current_resource_type,
+  recommended_resource_type,
+  implementation_effort,
+  restart_needed,
+  estimated_monthly_savings_after_discount as savings_amount
+from
+  aws_cost_optimization_recommendation
+where
+  implementation_effort = 'Low'
+  and restart_needed = false
+order by
+  savings_amount desc;
+```
+
+### Recommendations by Service Type
+
+Group recommendations by resource type to focus optimization efforts.
+
+```sql
+select
+  current_resource_type,
+  count(*) as recommendation_count,
+  round(sum(estimated_monthly_savings_after_discount), 2) as total_potential_savings,
+  round(avg(estimated_savings_percentage_after_discount), 2) as avg_savings_percentage
+from
+  aws_cost_optimization_recommendation
+group by
+  current_resource_type
+order by
+  total_potential_savings desc;
+```
+
+### Regional Optimization Opportunities
+
+Analyze cost-saving opportunities by AWS region.
+
+```sql
+select
+  region,
+  count(*) as recommendation_count,
+  round(sum(estimated_monthly_savings_after_discount), 2) as total_potential_savings,
+  round(avg(estimated_savings_percentage_after_discount), 2) as avg_savings_percentage
+from
+  aws_cost_optimization_recommendation
+group by
+  region
+order by
+  total_potential_savings desc;
+```
+
+## Example Configurations
+
+### Collect Recommendations from an S3 Bucket
```

Comment:
Please convert all example configuration H3 titles to use sentence case (see other table docs as an example).

The query titles (above) should remain title case.